src/types/navigation.tsx

```diff
@@ -25,17 +22,25 @@ export type GrantsStackParamList = {
 
 export type SearchStackParamList = {
   Audio: undefined;
-  Play: undefined;
 };
 
 export type RootStackScreenProps<T extends keyof RootStackParamList> =
   NativeStackScreenProps<RootStackParamList, T>;
 
 export type HomeStackScreenProps<T extends keyof HomeStackParamList> =
-  NativeStackScreenProps<HomeStackParamList, T>;
+  CompositeScreenProps<
```

Comment:
Would recommend adding a comment here to explain why we do this -- something like 

`// Note: we use composite screen props to support navigating to the 'Play' screen from any stack`